src/main/java/seedu/parking/logic/commands/FilterCommand.java

```diff
@@ -14,25 +14,23 @@
 import seedu.parking.model.carpark.CarparkContainsKeywordsPredicate;
 import seedu.parking.model.carpark.CarparkFilteringPredicate;
 
-
 /**
- * Filters car parks using to flags
+ * Filters the list of car parks returned by the previous find command with the use of flags
  */
 public class FilterCommand extends Command {
     public static final String COMMAND_WORD = "filter";
     public static final String COMMAND_ALIAS = "fi";
```

Comment:
Hey why is it still COMMAND_ALIAS. I have refactored all COMMAND_ALIAS to COMMAND_ABBREVIATION. this will break the abbreviation mechanism.